packages/cli-commands/src/commands/start/startServerInNewWindow.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ * @oncall react_native
+ */
+
+import type {ExecaPromise, SyncResult} from 'execa';
+
+import path from 'path';
+import fs from 'fs';
+import execa from 'execa';
+import {
+  CLIError,
+  logger,
+  resolveNodeModuleDir,
+} from '@react-native-community/cli-tools';
+
+export function startServerInNewWindow(
+  port: number,
+  terminal: string,
+  projectRoot: string,
+  reactNativePath: string,
+): SyncResult | ExecaPromise | CLIError | void {
+  /**
+   * Set up OS-specific filenames and commands
+   */
+  const isWindows = /^win/.test(process.platform);
+  const scriptFile = isWindows
+    ? 'launchPackager.bat'
+    : 'launchPackager.command';
```

Comment:
Not sure where to write this comment, but please make sure to check if the file permissions when building this `cli-commands` package the permissions for these files are correct. For reference: https://github.com/react-native-community/cli/pull/1904

Reply:
Thanks, I believe they are — but also I'm intending to squash in the recent removal of these files once merged in CLI :)

Reply:
Ah, yes! Many changes in that part of codebase recently 😅